### PR TITLE
When User is in Creator Hub and is on the Published NFT- Publish Button Should NOT be an option and only have One Pop Screen to be Directed to Pylons or IPFS #1376

### PR DESCRIPTION
**Pull request by Kamran-Khan-Gene**
1- Removed Publish button from published NFT in GridView.
2- Added NTF File Types Icons.
3- Added test cases to test NFT File Type Icons.

## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by Kamran-Khan-Gene** [comment]
@kjawadDeveloper2 kindly review my PR

File: easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart [hunk 1]
```diff
@@ -118,105 +118,113 @@ class NFTsListTile extends StatelessWidget {
...
+                            width: 50.0.w,
+                            height: 50.0.w,
+                            decoration: const BoxDecoration(
+                              color: EaselAppTheme.kLightGrey03,
+                            ),
+                          ),
+                          SizedBox(
+                            width: 10.w,
+                          ),
+                          Expanded(
```
Comment (kjawadDeveloper2) [inline easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart:180]: What this is for ? I see only empty list.

File: easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart [hunk 2]
```diff
@@ -118,105 +118,113 @@ class NFTsListTile extends StatelessWidget {
...
-                      return const IgnorePointer(child: SizedBox());
-                    },
-                  ),
-                ),
-              ) else const IgnorePointer(
-                child: SizedBox(),
+                          SizedBox(
+                            width: 10.w,
+                          ),
+                          InkWell(
```
Comment (kjawadDeveloper2) [inline easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart:201]: Why we have something that is not working?

File: easel/test/widget/file_type_icon_test.dart [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+import '../mock/mock_repository.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+  final repository = MockRepositoryImp();
+  GetIt.I.registerLazySingleton(() => CreatorHubViewModel(repository));
+
+  group("Test NFT File Type Icons", () {
+    testWidgets(
+      "Image Type NFT Icon Test",
```
Comment (kjawadDeveloper2) [inline easel/test/widget/file_type_icon_test.dart:20]: The description doesnt match the way the test description should be.

**Comment by kjawadDeveloper2** [comment]
@Kamran-Khan-Gene  Title should be small and precise.

File: easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart (hunk 2 above)
Comment (Kamran-Khan-Gene) [reply easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart:201]: this is the place holder of the image, i guess that's why there is no feature implementation. 

image

File: easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart (hunk 1 above)
Comment (Kamran-Khan-Gene) [reply easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart:180]: it is a place holder for cacheImages.

File: easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart (hunk 2 above)
Comment (kjawadDeveloper2) [reply easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart:201]: If there is no use remove it.

File: easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart (hunk 1 above)
Comment (kjawadDeveloper2) [reply easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart:180]: strange is there any better way?

**Comment by Kamran-Khan-Gene** [comment]
> @Kamran-Khan-Gene Title should be small and precise.

title of?

File: easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart (hunk 2 above)
Comment (Kamran-Khan-Gene) [reply easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart:201]: image


image


image

Comment (Kamran-Khan-Gene) [reply easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart:201]: @kjawadDeveloper2 kindly let me know what should I do.. should I change it all over the project?

Comment (kjawadDeveloper) [reply easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart:201]: I will prefer shimmer.

**Review by kjawadDeveloper2 — APPROVED** [review]

**Review by Amean — APPROVED** [review]

**Pull request merged**